lib/agent0/agent0/hyperdrive/interactive/chain.py

```diff
@@ -429,11 +429,14 @@ class Config(Chain.Config):
             Number of seconds to advance time for every mined block. Uses real time if None.
         chain_port: int
             The port to bind for the anvil chain. Will fail if this port is being used.
+        transaction_block_keeper: int
+            The number of blocks to keep transaction records for.
```

Comment:
Add a note that this flag is undocumented in anvil and we are not positive how it works.